packages/terra-status-view/src/StatusView.scss

```diff
@@ -0,0 +1,111 @@
+@import '~terra-mixins';
+
+:local {
+  .status-view {
+    height: 100%;
+    padding-bottom: var(--terra-status-view-padding-bottom, 20px);
+    padding-top: var(--terra-status-view-padding-top, 20px);
+  }
+
+  .status-view-inner {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+  }
+
+  .glyph {
+    flex: 0 0 auto;
+    padding-bottom: 20px;
+  }
+
+  .glyph > svg {
+    background-repeat: no-repeat;
+    background-size: contain;
+    height: 100px;
+    width: 100px;
+  }
+
+  .glyph > .no-data {
+    background: var(--terra-status-view-glyph-no-data, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48' class='is-bidi'><path d='M45.8 12c1.2 0 2.2 1 2.2 2.2v26.7c0 1.2-1 2.2-2.2 2.2H2.2C1 43 0 42 0 40.8V7.2C0 6 1 5 2.2 5H15c1.4.1 2.7.6 3.7 1.6l3.8 3.9c1 1 2.3 1.5 3.7 1.6l19.4-.1h.2z'/></svg>"));
```

Comment:
i feel like we should break SVGs out into their own file and import them here as SASS variables

Reply:
I want to stick to theming through css custom properties as much as possible. If we start doing theming in both Sass and css custom properties, it adds to theming complexity.